translate/api/Google.Cloud.Translate.V3.Samples/TranslateV3BatchTranslateText.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("LongRunningRequestPollUntilComplete", "translate_v3_batch_translate_text")
+
+using CommandLine;
+// [START translate_v3_batch_translate_text]
+using Google.Cloud.Translate.V3;
+using System;
+using System.Collections.Generic;
+
+namespace Google.Cloud.Translate.V3.Samples
+{
+    public class TranslateV3BatchTranslateText
```

Comment:
Prefer not to add these long prefixes "TranslateV3" to the name of the classes and files. It's redundant information and it hinders readability.